doc/INSTALL.md

```diff
@@ -65,8 +65,6 @@ gpg (GnuPG) 2.1.15
     $ pip3 install --user -e trezor-agent/agents/trezor
     ```
 
-    Read [these instructions](https://github.com/romanz/python-trezor#pin-entering) on how to enter your PIN with the PIN entry.
```

Comment:
I think I was originally going to use this as an lead in to installing a separate pin-entry, but the pin entry stuff is probably fine collected in a separate document and I don't think this line is particularly relevant by itself.